manager/integration/tests/common.py

```diff
@@ -2930,3 +2930,24 @@ def get_liveness_probe_spec(initial_delay=5, period=5):
     }
 
     return pod_liveness_probe_spec
+
+
+def wait_for_pod_remount(core_api, pod_name):
+    check_command = [
+        '/bin/sh',
+        '-c',
+        'ls /data/'
+    ]
+
+    for i in range(10):
```

Comment:
1. If we have this, we don't need `wait_for_pod_restart` since that's not the real case we want to check.
2. Retry time is not enough. Assuming the pod will restart multiple times during the period.